mimic/model/nova_objects.py

```diff
@@ -933,67 +931,6 @@ def request_action(self, http_action_request, server_id, absolutize_url):
         else:
             return dumps(bad_request("There is no such action currently supported", http_action_request))
 
-    # Server Images
-
-    def image_by_id(self, image_id):
-        """
-        Retrieve a :obj:`Image` object by its ID.
-        """
-        for image in self.image_store:
-            if image.image_id == image_id and image.status != u"DELETED":
-                return image
-
-    def image_by_name(self, image_name):
-        """
-        Retrieve a :obj:`Image` object by its ID.
-        """
-        for image in self.image_store:
-            if image.name == image_name:
-                return image
-
-    def _create_random_list_of_images(self):
-        """
-        Creates a list of images.
-        """
-        for each_image in random_image_list:
-            if not self.image_by_name(each_image['name']):
-                image = Image(image_id=each_image['id'], name=each_image['name'],
-                              distro=each_image['distro'], tenant_id=self.tenant_id)
-                self.image_store.append(image)
-
-    def list_server_image(self, include_details, absolutize_url):
-        """
-        Return a list of images with details.
-        """
-        self._create_random_list_of_images()
-        result = {
-            "images": [
-                image.brief_json(absolutize_url) if not include_details
-                else image.get_server_image_details_json(absolutize_url)
-                for image in self.image_store
-            ]
-        }
-        return dumps(result)
-
-    def get_image(self, http_get_request, image_id, absolutize_url):
-        """
-        Return a image object if one exists from the list `/images` api,
-        else creates and adds the image to the :obj: `images_store`.
-        If the `image_id` is listed in `mimic.canned_responses.mimic_presets`,
-        then will return 404.
-        """
-        if (
-            image_id in get_presets['servers']['invalid_image_ref'] or
-            image_id.endswith('Z')
```

Comment:
The image starts with Z has been removed above. This will cause tests to fail for our consumers.